samples/core/blur/main.cpp

```diff
@@ -132,24 +135,25 @@ int main(int argc, char* argv[])
 
             // Similar to dual_pass_subgroup_exchange_box_blur but with a gauss
             // kernel.
-            if (use_subgroup_exchange_relative)
+            if (use_subgroup_exchange_relative && opencl_version_2_0)
             {
                 std::cout
                     << "Dual-pass subgroup relative exchange Gaussian blur"
                     << std::endl;
-
-                blur.build_program("-D USE_SUBGROUP_EXCHANGE_RELATIVE ");
+                // cl_khr_subgroup_shuffle_relative requires OpenCL 2.0
+                blur.build_program(
+                    "-D USE_SUBGROUP_EXCHANGE_RELATIVE -cl-std=CL2.0 ");
```

Comment:
See comment above: there is no dependency between `cl_khr_subgroup_shuffle_relative` and OpenCL C 2.0.